renku/core/models/projects.py

```diff
@@ -183,3 +206,28 @@ def __iter__(self):
             self.Meta.model(data, client=self._client, collection=self)
             for data in self._client.api.list_projects()
         )
+
+
+prov = fields.Namespace('http://www.w3.org/ns/prov#')
+schema = fields.Namespace('http://schema.org/')
+
+
+class ProjectSchema(JsonLDSchema):
+    """Project Schema."""
+
+    class Meta:
+        """Meta class."""
+
+        rdf_type = [schema.Project, prov.Location]
+        model = Project
+        translate = {
```

Comment:
does calamus actually use the `translate` fields?

Reply:
It's not really needed and will be removed.